### ISSUE TITLE
Some or all JS files are not respected by potx

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop-contrib/potx#3]
I extracted the language template files of the `core` and the `profiles` directory of a Backdrop 1.6.3. site using the *potx* module via Drush.

When I compared these files at random with a language template file exported otherwise, I noticed that some (or all?) `.js` files weren't respected by *potx*.

As an example see `core/misc/backdrop.js` which seemingly wasn't interpreted by *potx* although there are relevant language strings in it, for instance `Backdrop.t("An AJAX HTTP error occurred.")`.

**Comment by serundeputy** [comment]
ok.  I'll check into this js integration. thanks!

**Comment by serundeputy** [comment]
replace `Drupal.t`.

**Comment by quicksketch** [comment]
I went ahead and merged my PR at https://github.com/backdrop-contrib/potx/pull/4. It gets the UI working.

- Enable potx module.
- Visit /admin/config/regional/translate/extract
- Select a module that has calls to either Drupal.t() or Backdrop.t() (both should be detected now).
- Export.
- Get a .pot file downloaded in the browser.

I noticed that while Backdrop.t() calls are now exported, they all are reported as line number "0". But that seems to be the current design of the module that it doesn't track line numbers in JS files. So it's not perfect but it's matching the Drupal module functionality.
